package.json

```diff
@@ -34,16 +34,8 @@
     "built/web/vs/editor/editor.main.js",
     "built/web/vs/language/typescript/lib/*.js",
     "built/web/vs/language/typescript/src/*.js",
-    "built/web/skillmap/css/*.css",
-    "built/web/skillmap/js/*.js",
-    "built/web/authcode/css/*.css",
-    "built/web/authcode/js/*.js",
-    "built/web/multiplayer/css/*.css",
-    "built/web/multiplayer/js/*.js",
-    "built/web/kiosk/css/*.css",
-    "built/web/kiosk/js/*.js",
-    "built/web/teachertool/css/*.css",
-    "built/web/teachertool/js/*.js",
+    "built/web/*/css/*.css",
+    "built/web/*/js/*.js",
```

Comment:
It'd be nice if we could update the npm-prepare.js script (gets called as a part of prepare at the bottom of this file) to also reference subwebapp.ts but I'm not sure if that's possible...

Reply:
done! I moved this config into a json file that gets used in npm-prepare.js, in the cli, and in the gulpfile

Reply:
Nice 😎